.github/workflows/ci.yml

```diff
@@ -81,22 +86,28 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: ${{ secrets.NODE_VERSION }}
-    - uses: actions/cache@v1
+    - name: Cache node modules
+      id: node-cache
+      uses: actions/cache@v2
       with:
-        path: ~/.npm
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        path: |
+          ~/cache
+          ~/.cache
+          !~/cache/exclude
+          **/node_modules
+        key: ${{ runner.os }}-node-${{ hashFiles('**/lockfiles') }}
         restore-keys: |
           ${{ runner.os }}-node-
-    - run: |
+    - name: Install dependencies
```

Comment:
Why is the rename required ?

Reply:
That was only to have a better descriptive name for that step:

Before:

![image](https://user-images.githubusercontent.com/2797052/98413521-5d94e680-203f-11eb-8304-af5760e57775.png)


After:

![image](https://user-images.githubusercontent.com/2797052/98413517-5b328c80-203f-11eb-915b-987736a94920.png)